utils/bin/lvim

```diff
@@ -3,4 +3,36 @@
 export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-$HOME/.local/share/lunarvim}"
 export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-$HOME/.config/lvim}"
 
+headless_update() {
+  nvim --cmd 'lua _G.__lvim_updating=true' -u "$LUNARVIM_RUNTIME_DIR/lvim/init.lua" --noplugin -n -i NONE --headless
```

Comment:
Would this quit at the end? maybe with an explicit `+'quitall'`, or some `on_complete` event.

Reply:
Please look at `updater.lua`